heka/meta/heka.yml

```diff
@@ -476,6 +476,7 @@ aggregator:
 {%- endif %}
 ceilometer_collector:
   decoder:
```

Comment:
decoder: will null if `ceilometer_collector.amqp_host` is not defined
which is bad